test/resources/json/model/api/paye.json

```diff
@@ -0,0 +1,33 @@
+{
+  "employments": [
+    {
+      "employmentId": "01318d7c-bcd9-47e2-8c38-551e7ccdfae3",
+      "startDate": "2016-01-21",
+      "endDate": "2017-01-01",
+      "payeReference": "paye-1",
+      "employerName": "employer-1"
+    },
+    {
+      "employmentId": "019f5fee-d5e4-4f3e-9569-139b8ad81a87",
+      "startDate": "2016-02-22",
+      "payeReference": "paye-2",
+      "employerName": "employer-2"
+    }
+  ],
+  "allowances": [
+    {
+      "allowanceId": "c9923a63-4208-4e03-926d-7c7c88adc7ee",
+      "iabdType": "payeType",
+      "amount": 12
+    }
+  ],
+  "benefits": {
+    "UUID": [
```

Comment:
Should this be a real UUID here? I think it represents a employmentId is this correct?

Reply:
Yes  It will be uuid of the employment

Reply:
I will update the test